tests/cli/fluvio_smoke_tests/e2e-basic.bats

```diff
@@ -211,6 +222,13 @@ teardown_file() {
     assert_success
 }
 
+@test "Consume zstd message" {
```

Comment:
We need to disable this test for the cases where CLI or SPU are `stable`. Similar to what we did on https://github.com/infinyon/fluvio/pull/3014/commits/e6a92054a6e823cb01f36d148cfca57b6d21064b

Reply:
Makes total sense!